compose/compose-file/index.md

```diff
@@ -752,6 +753,7 @@ services:
       update_config:
         parallelism: 2
         delay: 10s
+        order: stop-first
```

Comment:
Actually; perhaps this needs the `version: '3'` to be updated to `version: '3.4'`, otherwise the example won't work.

@londoncalling what's the usual approach to mention which version something was added in?

Reply:
Oh, good call - didn't even see the `version` header there 😉

Reply:
@thaJeztah I've updated the header, and added a support note (based on the wording used elsewhere in the file)

Reply:
Thanks!